### PR TITLE
added documenation for Badge-button-avatar

### PR DESCRIPTION
**Pull request by DevRaahul**
Added documentation page for Badge, Button, Avatar component.
Removed unwanted CSS code.
Implemented suggested changes from last PR.
Avatar screen - 
image
Badge Screen-
https://drive.google.com/file/d/10CuxUu3Rh_qUDFXcp8mJg88EnDKUYxQ1/view
Button screen -
image

### CODE REVIEW

File: Avatar/index.html [hunk 1]
```diff
@@ -6,22 +6,95 @@
...
+                    style="width: 100%; height: 400px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (NamrataRaikwar2002) [inline Avatar/index.html:90]: you should remove inline style.

File: Badge/index.html [hunk 2]
```diff
@@ -6,69 +6,137 @@
...
-        <div class="avatar-badge-container">
-            <div class="avatar avatar-lg">
-                <img loading="lazy" class="img-responsive img-round" src="../assets/images/logo_2.jpg" alt="logo">
-                <span class="status-badge badge-lg badge-online top-right"></span>
+    <div class="menu">
+        <header>
+            <div id="hamburger" class="hamburger-menu">
+                <span></span>
+                <span></span>
+                <span></span>
```
Comment (NamrataRaikwar2002) [inline Badge/index.html:20]: These elements are of no use, you can remove them.

File: Badge/index.html [hunk 3]
```diff
@@ -6,69 +6,137 @@
...
+                    style="width: 100%; height: 450px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (NamrataRaikwar2002) [inline Badge/index.html:104]: You can remove these inline style

File: Avatar/index.html [hunk 4]
```diff
@@ -6,22 +6,95 @@
...
-    <div class="container">
-        <div class="avatar avatar-lg">
-            <img loading="lazy" class="img-responsive img-round" src="../assets/images/logo_2.jpg" alt="logo">
-        </div>
-        <div class="avatar avatar-md">
-            <img loading="lazy" class="img-responsive img-round" src="../assets/images/logo_2.jpg" alt="logo">
-        </div>
-        <div class="avatar avatar-sm">
-            <img loading="lazy" class="img-responsive img-round" src="../assets/images/logo_2.jpg" alt="logo">
+<body style="overflow-y: hidden;">
```
Comment (Divya70) [inline Avatar/index.html:14]: remove inline styling

Comment (NamrataRaikwar2002) [inline Avatar/index.html:14]: Remove these inline styling.

**Review by NamrataRaikwar2002 — APPROVED** [review]
All looks good, some changes are needed.

File: Avatar/index.html (hunk 4 above)
Comment (DevRaahul) [reply Avatar/index.html:14]: removed

File: utility.css [hunk 5]
```diff
@@ -36,11 +36,48 @@
 
 /* code snippet class */
 .code-snipet-container {
-  box-sizing: border-box;
   width: 100%;
-  max-height: 300px;
+  max-height: 450px;
   padding: 1rem 0;
-  background-color: #fafafa;
```
Comment (Divya70) [inline utility.css:43]: use color variable

File: utility.css [hunk 6]
```diff
@@ -36,11 +36,48 @@
...
-  margin: 2rem 0;
```
Comment (Divya70) [inline utility.css:44]: why  you use margin property two times , is there any reason ?

File: Badge/index.html (hunk 2 above)
Comment (DevRaahul) [reply Badge/index.html:20]: i have used these span for hamburger icon so these tags are in use

**Review by Divya70 — APPROVED** [review]
suggested some chnages , rest is fine for me

File: utility.css (hunk 6 above)
Comment (DevRaahul) [reply utility.css:44]: these are the deleted props from css file

File: utility.css (hunk 5 above)
Comment (DevRaahul) [reply utility.css:43]: this style props is deleted from file

**Review by Sindhu983 — COMMENTED** [review]
Approved

**Pull request merged**